MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -327,6 +327,18 @@ private void StartThrottler(StreamState state, TranscodingJob transcodingJob)
 
         private bool EnableThrottling(StreamState state)
         {
+            var encodingOptions = ServerConfigurationManager.GetEncodingOptions();
+
+            // enable throttling when not using hardware acceleration
+            if (encodingOptions.HardwareAccelerationType == string.Empty)
+            {
+                return state.InputProtocol == MediaProtocol.File &&
+                       state.RunTimeTicks.HasValue &&
+                       state.RunTimeTicks.Value >= TimeSpan.FromMinutes(5).Ticks &&
```

Comment:
this feels somewhat wrong, as I thought that `RunTimeTicks` is for play position, not transcode... was I wrong?

Reply:
i just used what was there, tried it with some files and it looked like it worked

Reply:
Well, according to my search I was probably wrong. `state.RunTimeTicks` seems to represent total length of the item, so this check effectively says "enable throttling on items longer than 5 minutes".

I still don't like current implementation of the throttler, but we can give it a chance.

Reply:
well thats an ok reason to disable throttling i think? a 5min file should be transcoded pretty much instantly and while paused the ffmpeg process is using RAM